forc-plugins/forc-client/src/op/call/mod.rs

```diff
@@ -234,21 +242,57 @@ pub async fn call(cmd: cmd::Call) -> anyhow::Result<String> {
         }
     };
 
-    forc_tracing::println_action_green("receipts:", &format!("{:#?}", receipts));
-    forc_tracing::println_action_green("tx hash:", &tx_hash.to_string());
-    forc_tracing::println_action_green("result:", &result);
+    // print receipts
+    if show_receipts {
+        forc_tracing::println_action_green("receipts:", &format!("{:#?}", receipts));
```

Comment:
nit: Personally I think there is too much color in the output. Maybe the labels could be a color but the actual logs and tx hash are plain. Or the labels could just be bold.

Reply:
I have the same opinion as Sophie here.

Reply:
addressed: https://github.com/FuelLabs/sway/pull/6968/commits/9adc3a92e64da69de1c013b5fcf027a45c15c64c
example output:
<img width="713" alt="Screenshot 2025-03-01 at 12 14 46 AM" src="https://github.com/user-attachments/assets/e630ba0a-005f-40ca-8f7e-d889b48336a9" />